src/featuresToUnlock/unlockableAppIcons.ts

```diff
@@ -110,7 +114,15 @@ export const FiniliarIcon: UnlockableAppIcon = {
   key: 'finiliar',
   explainSheetType: 'finiliar_app_icon',
   network: Network.mainnet,
-  unlockingNfts: [FINILIAR_NFT_ADDRESS],
+  unlockingNfts: [FINI_NFT_ADDRESS, RAINBOW_FINI_NFT_ADDRESS],
+  unlockKey: 'finiliar_nft_app_icon',
+};
+
+export const FiniliarIconBase: UnlockableAppIcon = {
```

Comment:
currently can't support app icon w/ unlocking nfts on multiple networks, so just added a new `UnlockableAppIcon` for the same mmkv key, but unlocked by the base network fini collection